public/app/percona/add-instance/components/AddRemoteInstance/AddRemoteInstance.tools.tsx

```diff
@@ -53,3 +88,43 @@ export const getInstanceData = (instanceType: string, credentials: any): Instanc
 
   return instance;
 };
+
+export const getInstanceData = (instanceType: string, credentials: any): InstanceData => {
+  const extractCredentials = (credentials?: any): InstanceData => {
```

Comment:
Missing type for credentials. And if it's expected that this can be undefined tell that with `type | undefined`

Reply:
Left it for future refactoring